src/Bicep.Core/CoreResources.Designer.cs

```diff
@@ -10,8 +10,8 @@
 
 namespace Bicep.Core {
```

Comment:
This is a generated file - please could you revert the changes to it?

Reply:
Reverted all, but one change in ln 1. Not sure why it's complaining. VS IDE does some auto formatting. I tried copying it from a clean machine, still has a diff in ln 1